tests/assertions/toHaveBeenFetched.test.js

```diff
@@ -55,6 +55,25 @@ describe('toHaveBeenFetched', () => {
     configure({ defaultHost: '' })
   })
 
+  it('should check that the path has been called with custom host without protocol', async () => {
+    const path = '//other-domain.com/some/path/'
+    const expectedPath = '/some/path/'
+    const host = 'https://other-domain.com'
+    configure({
+      defaultHost: '/some-domain.com',
+    })
+    await fetch(new Request(path))
+    const { message } = await assertions.toHaveBeenFetched(expectedPath, {
+      host,
+    })
+
+    expect(message()).toBe(
+      '🌯 Wrapito: https://other-domain.com/some/path/ is called',
```

Comment:
Why is there protocol in this message, when it checks host without protocol?

Reply:
The protocol error depends on the `defaultHost` defined in the `configure`. The asserted url does not give us any trouble 👀